src/lib/canUseDom.js

```diff
@@ -0,0 +1,3 @@
+export default function canUseDom() {
+  return typeof window !== 'undefined' && window.document
+}
```

Comment:
Ah, I see that all the call-sites use this as a value/variable not a function:
```suggestion
export default !!(typeof window !== 'undefined' && window.document)
```

Reply:
Sorry yes, I should have picked up on that 🤦, I took the note from [the linter](https://eslint.org/docs/rules/no-implicit-coercion) to use Boolean here too ¯\_(ツ)_/¯